packages/h2-async/h2-async.0.6.1/opam

```diff
@@ -14,7 +14,7 @@ depends: [
   "dune" {>= "1.7"}
   "faraday-async"
   "async" {< "v0.14"}
-  "gluten-async"
+  "gluten-async" {< "0.3.0"}
```

Comment:
Side note: Even though 0.3.0 is already in opam-repository, this combination wasn’t possible anyway since `gluten-async.0.3.0` requires async >= v0.15